pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -155,6 +155,8 @@ type ResourceSelector struct {
 
 	// Name of the target resource.
 	// Default is empty, which means selecting all resources.
+	// For PropagationPolicy(ClusterPropagationPolicy), it cannot be empty
+	// if Preemption is "Always".
```

Comment:
I'd prefer to postpone adding this description(restriction) to ResourceSelector. 
This restriction is temporary and might be removed in the future.